deploy/manifests/scc.yaml

```diff
@@ -0,0 +1,37 @@
+# Create SCC for Gateway resources
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: nginx-ngf-admin
```

Comment:
Should we make this name configurable through helm values?

Reply:
I think it would be unnecessary as I'm not sure of any use cases for a user to configure this name to something separate. However, I'm open to change as I don't see any harm besides adding more stuff to our helm Values. (would also need to change the rbac to match)

Reply:
> I think it would be unnecessary as I'm not sure of any use cases for a user to configure this name to something separate

Though unlikely, the user could have another SCC named `nginx-ngf-admin`, or they may have a standard naming system that this name does not conform to.

Reply:
I think I have done this correctly, the results are in the conversation thread above.

Reply:
Commented separately, the value needs to be added to the RBAC template too otherwise looks good 👍🏼